recipe/conda_build_config.yaml

```diff
@@ -24,7 +24,6 @@ blas_impl:
 github_actions_labels:          # [linux or win]
 - cirun-openstack-cpu-2xlarge   # [linux]
 - cirun-openstack-gpu-2xlarge   # [linux]
-- cirun-openstack-gpu-2xlarge   # [linux]
```

Comment:
you need to delete both 2/3 if you want to use the current pinning (post revert of https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6910)

Reply:
just run all linux builds on a gpu machine?

Reply:
Yeah, that's what I wanted to avoid, but we're stuck with it for now.